src/apiary.apib

```diff
@@ -1649,12 +1646,10 @@ level `conversions` array.
 + One Of
     + commission_rate: 4
     + commission_value
-    + commission_value_cpc
```

Comment:
CPC is handled via a combination of params:

```
sale_type: cpc
commission_value: 0.10
```

You cannot send `sale_type` with with `commission_rate` though.